presto-main/src/test/java/com/facebook/presto/catalogserver/TestCatalogServerResponse.java

```diff
@@ -60,28 +60,32 @@ public void setup()
     public void testSchemaExists()
             throws Exception
     {
-        boolean schemaExists = testingCatalogServerClient.schemaExists(null, null, null);
+        CatalogServerClient.MetadataEntry<Boolean> metadataEntry = testingCatalogServerClient.schemaExists(null, null, null);
+        boolean schemaExists = metadataEntry.getReturnValue();
         boolean actualSchemaExists = false;
 
         assertEquals(schemaExists, actualSchemaExists);
+        assertEquals(metadataEntry.getIsCacheHit(), true);
```

Comment:
`assertTrue()` and there are a few other instances in this file.

Reply:
Done